src/sentry/eventstore/snuba/backend.py

```diff
@@ -1,10 +1,51 @@
 from __future__ import absolute_import
 
+from datetime import datetime
 
+from sentry.utils import snuba
+from sentry.models import SnubaEvent
 from sentry.eventstore.base import EventStorage
 
+DEFAULT_START = datetime.utcfromtimestamp(0)  # will be clamped to project retention
+DEFAULT_END = datetime.utcnow()  # will be clamped to project retention
+DEFAULT_ORDERBY = ['-timestamp', '-event_id']
+DEFAULT_LIMIT = 100
+DEFAULT_OFFSET = 0
+
 
 class SnubaEventStorage(EventStorage):
     """
     Eventstore backend backed by Snuba
     """
+
+    def get_event_by_id(self, project_id, event_id, additional_columns=None):
+        """
+        Get an event given a project ID and event ID
+        Returns None if an event cannot be found
+        """
+        cols = self.__get_columns(additional_columns)
+
+        result = snuba.raw_query(
```

Comment:
Could you please rely on the SnubaEvent code for this for now ?
The reasoning is that we will have SnubaEvent.objects.get_event and SnubaEvent.get_event and eventstore.get_event_by_id around together for a while. Having all three depending on raw_query opens the door to inconsistency. (What if someone makes a change to the default timestamp on Snuba.get_event and forgets about this one ? ).
If the event fetch code path is the same we guarantee consistency, and when everyone will depend on eventstore, we will be able to simply copy paste the code here and remove the method from SnubaEvent.

Reply:
Updated to just return SnubaEvent.get_event